core/lib/zksync_core/src/api_server/web3/mod.rs

```diff
@@ -119,6 +122,7 @@ impl Namespace {
         Namespace::Zks,
         Namespace::En,
         Namespace::Pubsub,
+        Namespace::Snapshots,
```

Comment:
Just to understand better: Do we *want* to enable this namespace for every node that doesn't configure namespaces manually (see e.g. [relevant EN config logic](https://github.com/matter-labs/zksync-era/blob/7c992323a31baafb882f6ebb448359e0635269b2/core/bin/external_node/src/config/mod.rs#L338-L342))? IMO, it should be an opt-in; e.g., for ENs exposing this namespace makes little sense. Perhaps, we should rename `NON_DEBUG` into something like `DEFAULT` to clearly signal its purpose. cc @RomanBrodetski

Reply:
hmmm actually we might want to expose it - and jsut respond that we don't have any snapshots generated. In the future nothing prevents EN operators to generate snapshots as well